libsignal-service/src/cipher.rs

```diff
@@ -177,6 +182,63 @@ impl ServiceCipher {
             device as i32,
         ));
     }
+
+    pub fn encrypt(
+        &self,
+        address: &ProtocolAddress,
+        unindentified_access: Option<bool>,
+        content: &[u8],
+    ) -> Result<OutgoingPushMessage, ServiceError> {
+        if unindentified_access.is_some() {
+            unimplemented!("unidentified access is not implemented");
+        } else {
+            let get_padded_message_length = |len: usize| {
+                let message_length_with_terminator = len + 1;
+                let mut message_part_count =
+                    message_length_with_terminator / 160;
+                if message_length_with_terminator % 160 != 0 {
+                    message_part_count += 1;
+                }
+
+                message_part_count * 160
+            };
```

Comment:
Maybe we should use [`block-padding`'s `Iso7816`](https://docs.rs/block-padding/0.2.1/block_padding/enum.Iso7816.html) for this, and replace the one in `decrypt` too.